src/utils/sanitizeContent.js

```diff
@@ -0,0 +1,18 @@
+import DOMpurify from "isomorphic-dompurify";
+
+export const sanitizeContent = (data, props) => {
+  const sanitizedContent = DOMpurify.sanitize(data.content);
+  
+  let chunk = { ...data, content: sanitizedContent };
+  let tokens = chunk.content.match(/\{{(.*?)\}}/g);
+
+  let parsedChunk = chunk.content;
+
+  if (tokens !== null) {
```

Comment:
How bout not making it stict comparison like `tokens != null` to catch `undefined` values?